modules/acknowledgements/jsx/columnFormatter.js

```diff
@@ -0,0 +1,17 @@
+/* exported formatAcknowledgementsColumn */
+
+/**
+ * Modify behaviour of specified column cells in the Data Table component
+ * @param {string} column - column name
+ * @param {string} cell - cell content
+ * @param {arrray} rowData - array of cell contents for a specific row
+ * @param {arrray} rowHeaders - array of headers for the table
+ * @return {*} a formated table cell for a given column
+ */
+function formatAcknowledgementsColumn(column, cell, rowData, rowHeaders) {
+  return <td>{cell}</td>;
+}
+
+window.formatAcknowledgementsColumn = formatAcknowledgementsColumn;
```

Comment:
This is not necessary if you export

Reply:
I can only delete "export default formatAcknowledgementsColumn;".
when I delete "window.formatAcknowledgementsColumn = formatAcknowledgementsColumn;",
the react table do not show up.

Reply:
I see. we sill fix this later when the ReactDOM.render will be removed from the templates.